src/JitInterface/src/ThunkGenerator/corinfo.h

```diff
@@ -691,6 +691,8 @@ enum CorInfoHelpFunc
 
     CORINFO_HELP_JIT_REVERSE_PINVOKE_ENTER, // Transition to cooperative mode in reverse P/Invoke prolog, frame is the first argument
     CORINFO_HELP_JIT_REVERSE_PINVOKE_EXIT,  // Transition to preemptive mode in reverse P/Invoke epilog, frame is the first argument
+
+    CORINFO_HELP_GVMLOOKUP_FOR_SLOT,        // Resolve a generic virtual method target from this pointer and runtime method handle 
```

Comment:
We should also add this to https://github.com/dotnet/corert/blob/master/src/JitInterface/src/CorInfoHelpFunc.cs